server/src/utils/media.ts

```diff
@@ -394,6 +394,35 @@ export class VP9Config extends BaseConfig {
   }
 }
 
+export class AV1Config extends BaseConfig {
+  getPresetOptions() {
+    const speed = this.getPresetIndex() + 4; // Use 4 as slowest, giving us an effective range of 4-12 which is far more useful than 0-8
+    if (speed >= 0) {
+      return [`-preset ${speed}`];
+    }
+    return [];
+  }
+
+  getBitrateOptions() {
+    const bitrates = this.getBitrateDistribution();
+    const svtparams = [];
+    if (this.config.threads > 0) {
+      svtparams.push(`lp=${this.config.threads}`)
+    }
+    if (bitrates.max > 0) {
+      svtparams.push(`mbr=${bitrates.max}${bitrates.unit}`)
+    }
+    return [
+      `-crf ${this.config.crf}`,
+      `-svtav1-params ${svtparams.join(':')}`
```

Comment:
You should only add this if `svtparams` isn't empty, else it will be an invalid command since `-svtav1-params` will expect an argument.

Reply:
Ah yeah that was an oversight, fixed.

Reply:
Oh damn, totally missed that and forgot to test that case, fixed